tests/test_wheel.py::test_override_tags_in_wheel_filename fails on pypy3.8

```
$ pypy3 -m pytest
========================================================= test session starts =========================================================
platform linux -- Python 3.8.12[pypy-7.3.7-final], pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /tmp/pdm-pep517
collected 50 items                                                                                                                    

tests/test_api.py .......x........                                                                                              [ 32%]
tests/test_file_finder.py ............                                                                                          [ 56%]
tests/test_metadata.py ............                                                                                             [ 80%]
tests/test_validator.py ........                                                                                                [ 96%]
tests/test_wheel.py F.                                                                                                          [100%]

============================================================== FAILURES ===============================================================
________________________________________________ test_override_tags_in_wheel_filename _________________________________________________

    def test_override_tags_in_wheel_filename() -> None:
        project = FIXTURES / "projects/demo-cextension"
        builder = wheel.WheelBuilder(
            project,
            config_settings={"--py-limited-api": "cp36", "--plat-name": "win_amd64"},
        )
>       assert builder.tag == "cp36-abi3-win_amd64"
E       AssertionError: assert 'pp38-pypy38_pp73-win_amd64' == 'cp36-abi3-win_amd64'
E         - cp36-abi3-win_amd64
E         + pp38-pypy38_pp73-win_amd64

tests/test_wheel.py:11: AssertionError
========================================================== warnings summary ===========================================================
pdm/pep517/_vendor/cerberus/errors.py:156
  /tmp/pdm-pep517/pdm/pep517/_vendor/cerberus/errors.py:156: DeprecationWarning: invalid escape sequence '*'
    """

pdm/pep517/_vendor/cerberus/errors.py:184
  /tmp/pdm-pep517/pdm/pep517/_vendor/cerberus/errors.py:184: DeprecationWarning: invalid escape sequence '*'
    """

tests/test_api.py::test_build_single_module
  /tmp/pdm-pep517/pdm/pep517/base.py:152: PDMDeprecatedWarning: `version` in [project] no longer supports dynamic filling. Move it to [tool.pdm] or change it to static string.
  It will raise an error in the next minor release.
    meta_version = self.meta.version

tests/test_api.py::test_build_single_module
  /tmp/pdm-pep517/pdm/pep517/base.py:347: PDMDeprecatedWarning: `version` in [project] no longer supports dynamic filling. Move it to [tool.pdm] or change it to static string.
  It will raise an error in the next minor release.
    version=meta.version or "UNKNOWN",

tests/test_api.py::test_build_single_module
  /tmp/pdm-pep517/pdm/pep517/sdist.py:94: PDMDeprecatedWarning: `version` in [project] no longer supports dynamic filling. Move it to [tool.pdm] or change it to static string.
  It will raise an error in the next minor release.
    self.meta._metadata["version"] = self.meta.version

tests/test_metadata.py::test_parse_module
  /tmp/pdm-pep517/tests/test_metadata.py:13: PDMDeprecatedWarning: `version` in [project] no longer supports dynamic filling. Move it to [tool.pdm] or change it to static string.
  It will raise an error in the next minor release.
    assert metadata.version == "0.1.0"

-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================================================= short test summary info =======================================================
FAILED tests/test_wheel.py::test_override_tags_in_wheel_filename - AssertionError: assert 'pp38-pypy38_pp73-win_amd64' == 'cp36-abi3...
========================================= 1 failed, 48 passed, 1 xfailed, 6 warnings in 6.86s =========================================
```

--- a/tests/test_wheel.py
+++ b/tests/test_wheel.py
@@ -1,7 +1,12 @@
+import sys
+
+import pytest
+
 from pdm.pep517 import wheel
 from tests import FIXTURES
 
 
+@pytest.mark.skipif(hasattr(sys, "pypy_version_info"), reason="invalid on PyPy3")
 def test_override_tags_in_wheel_filename() -> None:
     project = FIXTURES / "projects/demo-cextension"
     builder = wheel.WheelBuilder(

